setup.php

```diff
@@ -185,45 +193,45 @@ function plugin_init_example() {
    $PLUGIN_HOOKS['stats']['example'] = ['stat.php'       => 'New stat',
                                         'stat.php?other' => 'New stats 2',];
 
-   $PLUGIN_HOOKS['post_init']['example'] = 'plugin_example_postinit';
+   $PLUGIN_HOOKS[Hooks::POST_INIT]['example'] = 'plugin_example_postinit';
 
    $PLUGIN_HOOKS['status']['example'] = 'plugin_example_Status';
 
    // CSRF compliance : All actions must be done via POST and forms closed by Html::closeForm();
-   $PLUGIN_HOOKS['csrf_compliant']['example'] = true;
+   $PLUGIN_HOOKS[Hooks::CSRF_COMPLIANT]['example'] = true;
 
-   $PLUGIN_HOOKS['display_central']['example'] = "plugin_example_display_central";
-   $PLUGIN_HOOKS['display_login']['example'] = "plugin_example_display_login";
-   $PLUGIN_HOOKS['infocom']['example'] = "plugin_example_infocom_hook";
+   $PLUGIN_HOOKS[Hooks::DISPLAY_CENTRAL]['example'] = "plugin_example_display_central";
+   $PLUGIN_HOOKS[Hooks::DISPLAY_LOGIN]['example'] = "plugin_example_display_login";
+   $PLUGIN_HOOKS[Hooks::INFOCOM]['example'] = "plugin_example_infocom_hook";
 
    // pre_show and post_show for tabs and items,
-   // see PluginExampleShowtabitem class for implementation explanations
-   $PLUGIN_HOOKS['pre_show_tab']['example']     = ['PluginExampleShowtabitem', 'pre_show_tab'];
-   $PLUGIN_HOOKS['post_show_tab']['example']    = ['PluginExampleShowtabitem', 'post_show_tab'];
-   $PLUGIN_HOOKS['pre_show_item']['example']    = ['PluginExampleShowtabitem', 'pre_show_item'];
-   $PLUGIN_HOOKS['post_show_item']['example']   = ['PluginExampleShowtabitem', 'post_show_item'];
+   // see GlpiPlugin\Example\Showtabitem class for implementation explanations
+   $PLUGIN_HOOKS[Hooks::PRE_SHOW_TAB]['example']     = [Showtabitem::class, 'pre_show_tab'];
```

Comment:
I personaly have preference for class keyword but try to be consistent (`::getType()` or `::class` everwhere)

Reply:
switching to ::class everywhere, except for expressions like ```$item::getType()```